.eslintignore

```diff
@@ -0,0 +1 @@
+system-tests/projects/no-specs/src/Invalid.jsx
```

Comment:
I added this because ESLint wouldn't let me commit with this file because it has a syntax error, but the error is intentional for testing

Reply:
Does `/*eslint-disable */` not work?

Reply:
Updated this in https://github.com/cypress-io/cypress/pull/24982/commits/1670c94cfc0816bec0a99da012788589222a704a